addons/web/static/src/legacy/xml/colorpicker.xml

```diff
@@ -16,23 +16,23 @@
         </div>
         <div t-if="widget.options.colorPreview" class="o_color_preview mb-1 w-100 p-2"/>
         <div class="o_color_picker_inputs d-flex justify-content-between mb-2">
-            <t t-set="input_classes" t-value="'p-0 border-0 text-center text-monospace bg-transparent text-white'" />
+            <t t-set="input_classes" t-value="'p-0 border-0 text-center text-monospace bg-transparent o_colorpicker_text'" />
 
-            <div class="o_hex_div bg-black-50 px-1 d-flex align-items-baseline">
+            <div class="o_hex_div o_colorpicker_bg_50 border-left-0 px-1 d-flex align-items-baseline">
                 <input type="text" t-attf-class="o_hex_input {{input_classes}}" data-color-method="hex" size="1"/>
-                <span class="flex-grow-0 ml-1 text-white-50">hex</span>
+                <span class="flex-grow-0 ml-1 o_colorpicker_text_50">hex</span>
             </div>
-            <div class="o_rgba_div bg-black-50 px-1 d-flex align-items-baseline">
+            <div class="o_rgba_div o_colorpicker_bg_50 px-1 d-flex align-items-baseline">
```

Comment:
Good change, I just added an entry in my todo list to fix that today :+1: ... but that's because I was working on a merge of the new colorpicker widget, I guess your work should be based on it :grimacing: (r+'ed one hour ago: https://github.com/odoo/odoo/pull/67545)

Reply:
Rebased without any visible issue, the "css variables system" is holding fine

Reply:
Beware that we introduced a *new* bg-black-50 so even if you rebase over the unmerged work worked flawlessly you may need to change the new bg-black-50 (hopefully, the PR is being merged, was not merged yesterday because of a runbot "security" check).